app/models/event.rb

```diff
@@ -0,0 +1,3 @@
+class Event < ApplicationRecord
+	has_and_belongs_to_many :members
```

Comment:
I'm guessing this sets up a relation to members through some wizardry? Does it assume it connects through event_members?

Reply:
Yes, in Rails we can define a many-to-many relationship by stating `has_and_belongs_to_many` but this means the table that they join with (`events_members`) can not have a primary key or really any other info. The other way to do a many-to-many is to do a `has_many` and a `has_many <thing>, through: <join_table>`. We do this for `Accomplishments`.